package-lock.json

```diff
@@ -6,11 +6,12 @@
   "packages": {
     "": {
       "name": "roo-cline",
-      "version": "2.1.18",
+      "version": "2.1.18"
```

Comment:
Sorry I missed this @Premshay can you rebuild your `package-lock.json` too so it has the correct version number?